packages/server-clear-thought/index.ts

```diff
@@ -1380,66 +1346,64 @@ server.setRequestHandler(ListToolsRequestSchema, async () => ({
 }));
 
 server.setRequestHandler(CallToolRequestSchema, async (request) => {
-  switch (request.params.name) {
-    case "sequentialthinking": {
-      const result = thinkingServer.processThought(request.params.arguments);
-      return {
-        content: [{ type: "application/json", text: JSON.stringify(result, null, 2) }]
-      };
-    }
-    case "mentalmodel": {
-      const result = modelServer.processModel(request.params.arguments);
-      return {
-        content: [{ type: "application/json", text: JSON.stringify(result, null, 2) }]
-      };
-    }
-    case "debuggingapproach": {
-      const result = debuggingServer.processApproach(request.params.arguments);
-      return {
-        content: [{ type: "application/json", text: JSON.stringify(result, null, 2) }]
-      };
-    }
-    case "collaborativereasoning": {
-      const result = collaborativeReasoningServer.processCollaborativeReasoning(request.params.arguments);
-      return {
-        content: [{ type: "application/json", text: JSON.stringify(result, null, 2) }]
-      };
-    }
-    case "decisionframework": {
-      const result = decisionFrameworkServer.processDecisionFramework(request.params.arguments);
-      return {
-        content: [{ type: "application/json", text: JSON.stringify(result, null, 2) }]
-      };
-    }
-    case "metacognitivemonitoring": {
-      const result = metacognitiveMonitoringServer.processMetacognitiveMonitoring(request.params.arguments);
-      return {
-        content: [{ type: "application/json", text: JSON.stringify(result, null, 2) }]
-      };
-    }
-    case "scientificmethod": {
-      const result = scientificMethodServer.processScientificMethod(request.params.arguments);
-      return {
-        content: [{ type: "application/json", text: JSON.stringify(result, null, 2) }]
-      };
-    }
-    case "structuredargumentation": {
-      const result = argumentationServer.processStructuredArgumentation(request.params.arguments);
-      return {
-        content: [{ type: "application/json", text: JSON.stringify(result, null, 2) }]
-      };
-    }
-    case "visualreasoning": {
-      const result = visualReasoningServer.processVisualReasoning(request.params.arguments);
-      return {
-        content: [{ type: "application/json", text: JSON.stringify(result, null, 2) }]
-      };
-    }
-    default:
+  function fetch(server: { process: (args: unknown) => unknown, name?: string }, args: unknown) {
+    const result = server.process(args);
```

Comment:
renaming all the Server response methods `process` allows me to execute them generically here